pages/login.php

```diff
@@ -128,5 +132,24 @@
 			
 	</form>
 
+<?php include("../include/footer.inc.php") ?>
+
+
+<script>
+function box1()
```

Comment:
can you put this in a separate `js` file and link it here?

Reply:
done